### DISCUSSION TITLE
fa/app/How-to-install-Hiddify-app/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4189] [Content Help کمک به اصلاح محتوا]
# fa/app/How-to-install-Hiddify-app/

The proxy manager platform.

https://hiddify.com/fa/app/How-to-install-Hiddify-app/

**Comment by pejooo** [comment]


**Reply by TomCat-Hub** [reply]
درود بر شما دوست عزیز
وقت بخیر

خیلی ممنونم که کامنت گذاشتید و سوال خودتون رو با ما به اشتراک گذاشتید.
خیر، متاسفانه این امکان فراهم نیست. هیدیفای منیجر فقط و فقط روی سرور لینوکسی با سیستم عامل اوبونتو نسخه 22.04 نصب میشود.

لطفا اگر سوال بیشتری داشتید و یا نیاز به راهنمایی دقیق تری داشتید در گروه تلگرامی ما پیام بفرستید تا دوستان شما رو راهنمایی کنند.

آدرس گروه تلگرامی هیدیفای:
https://t.me/hiddify_board

باتشکر از توجه شما
مانا باشید!

**Comment by aiilvliir** [comment]
از همه دوستانی که برای هیدیفای زحمت میکشند ممنونم هیدیفای بی نظیره طیف گسترده ای از پروتکل ها پوشش میده و همچنین کاربر برای پیکربندی برنامه کاملا دستش بازه تنظیم که باشه خبری از مشکلات رایج در فیلترشکن های معمول در اون دیده نمیشه ممنون از همه دوستانی که  در این راه رایگان و بدون چشم داشت زحمت میکشن

**Comment by MMng57** [comment]
Большое спасибо разработчикам! Можно ли здесь дать ссылку на установку для АндроидТV?

**Reply by Kurbanoff1984** [reply]
дай пожалуйста если можно, буду очень признателен)

**Comment by ghost** [comment]


**Reply by parsarazor** [reply]
you can change the permission by running command below:
chmod +x *filename

**Comment by xxxxxxx-xxxxxxx** [comment]
Please, help with…
I can't turn on Hiddify from the iPhone control centre (only from the app). For convenience, can you add this?

**Comment by noneside** [comment]
can i use this in jeson? it is ubutun20.04 platform ,maybe it too old?

**Comment by Lisochkadew69** [comment]
У меня проблема. установила приложение запускаю но оно тупо само по себе закрывается...... что сделать надо?
